wicker/core/datasets.py

```diff
@@ -160,7 +161,10 @@ def __init__(
         :param data_service: S3DataService instance to use, defaults to None which uses default initializations
         :param filelock_timeout_seconds: number of seconds after which to timeout on waiting for downloads,
             defaults to FILE_LOCK_TIMEOUT_SECONDS
-        :param treat_objects_as_bytes: If set, don't try to decode ObjectFields and keep them as binary data.
+        :param treat_objects_as_bytes: If set, don't try to decode ObjectFields and keep them as binary data
+        :param filters: Rows which do not match the filter predicate will be removed, defaults to None
```

Comment:
Minor: use an affirmative sentence for lower "mental load" on the reader/user of the library, e.g.:

```suggestion
        :param filters: Only returns rows which match the filter. Defaults to None, i.e., returns all rows.
```